en/common/performance.mdown

```diff
@@ -196,8 +240,20 @@ query.whereKey("state", containedIn: ["SignedUp", "Verified"])
 ```common-java
 query.whereContainedIn("state", Arrays.asList("SignedUp", "Verified"));
 ```
+```common-csharp
+var states = new[] { "SignedUp", "Verified" };
```

Comment:
These 4 lines of code can be simplified to for consistency:

query. WhereContainedIn("state", new[] { "SignedUp", "Verified" });